WebDriverAgentLib/Utilities/FBConfiguration.h

```diff
@@ -19,6 +19,9 @@ NS_ASSUME_NONNULL_BEGIN
 /*! If set to YES will ask TestManagerDaemon for element visibility */
 @property (class, nonatomic, assign) BOOL shouldUseTestManagerForVisibilityDetection;
 
+/* The maximum typing frequency for all typing activities */
```

Comment:
I have no idea, to tell you the truth. `_XCT_sendString` is an internal method to XCTest and ill-documented.